Automattic/wp-calypso
# Help Center: Adjust spinner position and margin in send message input

Fixes https://github.com/Automattic/dotcom-forge/issues/9682

## Proposed Changes

* Fix Odie message input spinner vertical alignment

## Why are these changes being made?


* Spinner alignment is not correct after redesign changes.

## Testing Instructions



* Checkout branch or use live link.
* Navigate to `/home?flags=help-center-experience` and open the Help Center.
* Open a new or existing conversation.
* Send a message and verify the spinner in the message input field is vertically aligned.

## Pre-merge Checklist
